utils/extract_license_edition.sh

```
@@ -0,0 +1,55 @@
+  return 0
+}
+
+
+
+# Execute the python snippet: prefer local python3, fallback to docker
+
+if command -v python3 >/dev/null 2>&1; then
+extractLicense
+
+else
+  printf '%s' "$jwt" | docker run --rm -i python:3.12-slim python3 - <<PY
```

Shellcheck (0.10.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)